test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,10 +36,34 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        //empty list
+        assertNoNull();
+
+        //all objects are null
+        assertThereIsNull(null, null);
+        assertThereIsNull(null, null, null);
+
+        //there exists a null object
+        assertThereIsNull(null, "a", "b", null);
+        assertThereIsNull("abc", null);
+        assertThereIsNull("abc", "", "abc", null);
+        assertThereIsNull("", "abc", "a", null);
+        assertThereIsNull(null, new Integer(1));
+        assertThereIsNull(null, 1, new Integer(1));
+
+        //there is no null objects
+        assertNoNull("abc", "ab", "a");
+        assertNoNull(1, 2);
+        assertNoNull(1, 2, 3);
+    }
+
+    private void assertThereIsNull(Object... objects) {assertTrue(Utils.isAnyNull(objects));}
```

Comment:
Coding standard violation. `assertTrue(Utils.isAnyNull(objects));` should be at the next line.